src/test/java/org/jabref/model/entry/MonthTest.java

```diff
@@ -100,4 +100,36 @@ public void parseReturnsEmptyOptionalForInvalidInput() {
     public void parseReturnsEmptyOptionalForEmptyInput() {
         assertEquals(Optional.empty(), Month.parse(""));
     }
+    
+    @Test
+	public void parseCorrectlyByShortNameGerman() {
+		assertEquals(Optional.of(Month.JANUARY), Month.parse("Jan"));
+		assertEquals(Optional.of(Month.FEBRUARY), Month.parse("Feb"));
+		assertEquals(Optional.of(Month.MARCH), Month.parse("M�r"));
+		assertEquals(Optional.of(Month.APRIL), Month.parse("Apr"));
+		assertEquals(Optional.of(Month.MAY), Month.parse("Mai"));
+		assertEquals(Optional.of(Month.JUNE), Month.parse("Jun"));
+		assertEquals(Optional.of(Month.JULY), Month.parse("Jul"));
+		assertEquals(Optional.of(Month.AUGUST), Month.parse("Aug"));
+		assertEquals(Optional.of(Month.SEPTEMBER), Month.parse("Sep"));
+		assertEquals(Optional.of(Month.OCTOBER), Month.parse("Okt"));
+		assertEquals(Optional.of(Month.NOVEMBER), Month.parse("Nov"));
+		assertEquals(Optional.of(Month.DECEMBER), Month.parse("Dez"));
+	}
+
+	@Test
+	public void parseCorrectlyByFullNameGerman() {
+		assertEquals(Optional.of(Month.JANUARY), Month.parse("Januar"));
+		assertEquals(Optional.of(Month.FEBRUARY), Month.parse("Februar"));
+		assertEquals(Optional.of(Month.MARCH), Month.parse("M�rz"));
```

Comment:
Please also add a test for `Maerz`, which is returned by the Stuttgart System: http://www2.informatik.uni-stuttgart.de/zdi/buecherei/NCSTRL_listings/FAK/index.html

Reply:
I also put in an if-statement for checking for `Maerz` in the method `parseGermanShortMonth`, because I was not sure if the Locale checks for this.